### PR TITLE
Adding Redis

### PR DESCRIPTION
**Pull request by Nyholm**
I was too quick merging #51

### CODE REVIEW

**Review by shouze — APPROVED** [review]

**Comment by Nyholm** [comment]
Oh no. Can you see why Redis is failing?

**Comment by shouze** [comment]
> Oh no. Can you see why Redis is failing?

Yes this is probably because base image used for the build contains AND load redis by default I guess. So at the moment I guess we need to find a way to unload/avoid loading it. Let me find out a solution.

File: layers/redis/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION AS ext
+
+RUN pecl install redis
```
Comment (shouze) [inline layers/redis/Dockerfile:4]: That's it, so at the moment till redis ext gets removed from base image have to add force option:
```suggestion
RUN pecl install --force redis
```

**Comment by shouze** [comment]
@Nyholm ok so now... it's mongo's turn! I've made the same patch https://github.com/Nyholm/extra-php-extensions/pull/1

**Review by shouze — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you

**Comment by shouze** [comment]
> Thank you

You're welcome!

**Pull request merged**